### ISSUE TITLE
Deal with large content in a Page

### ISSUE THREAD

**Issue by fabien0102** [contiamo/operational-ui#810]
### Bug description

![image](https://user-images.githubusercontent.com/1761469/47641707-0cf07600-db67-11e8-84d2-1c8153e19dbb.png)

If we have large content as logs, the Page is not fit to the window anymore. For now I need to horizontal scroll to see my log, this is not a good UX. Ideally every children of the Page should be `max-width: 100%` but of course, with grid and css, this problem is not so simple 😁

**Comment by peterszerzo** [comment]
The following markup ensures that large content within grid cells won't overflow the layout and that grid boundaries will be consistently respected (abstractions not included):
```
<div style={{ gridTemplateColumns: "...", gridTemplateRows: "..." }}>
  <div style={{ position: "absolute", top: 0, left: 0, right: 0, bottom: 0, overflow: "auto" }}>
    {/* content for grid cell 1 */}
  </div>
  <div style={{ position: "absolute", top: 0, left: 0, right: 0, bottom: 0, overflow: "auto" }}>
    {/* content for grid cell 2 */}
  </div>
  <div style={{ position: "absolute", top: 0, left: 0, right: 0, bottom: 0, overflow: "auto" }}>
    {/* content for grid cell 3 */}
  </div>
</div>
```
I've used this across Pantheon query editors and have been generally very happy with them. I can't find the source/blog post that outlined it, but I'll follow up if I do.

**Comment by TejasQ** [comment]
To be honest, I find the absolute positioning a little ambiguous while trying to discern intent. Why? Why does a max-width with overflow rules not suffice?

**Comment by TejasQ** [comment]
I'll push up a PR to make my suggestion a little more concrete.

**Comment by fabien0102** [comment]
Closed by #825
